[datadog_integration_gcp_sts] Add IsPerProjectQuotaEnabled field (#2999)
DataDog/terraform-provider-datadog: docs/resources/integration_gcp_sts.md

- `cloud_run_revision_filters` (Set of String) Tags to filter which Cloud Run revisions are imported into Datadog. Only revisions that meet specified criteria are monitored.
- `host_filters` (Set of String) Your Host Filters.
- `is_cspm_enabled` (Boolean) Whether Datadog collects cloud security posture management resources from your GCP project. If enabled, requires `resource_collection_enabled` to also be enabled.

> - `is_per_project_quota_enabled` (Boolean) When enabled, Datadog applies the `X-Goog-User-Project` header, attributing Google Cloud billing and quota usage to the project being monitored rather than the default service account project.

Reviewer: ```suggestion
- `is_per_project_quota_enabled` (Boolean) When enabled, Datadog includes the `X-Goog-User-Project` header to attribute Google Cloud billing and quota usage to the monitored project instead of the default service account project.
```
